docs/source/apps/proj.rst

```diff
@@ -192,18 +192,18 @@ The following script
 
 .. code-block:: console
 
-    proj +proj=utm +lon_0=112w +ellps=clrk66 -r <<EOF
+    proj +proj=utm +zone=12 -r <<EOF
```

Comment:
```suggestion
    proj +proj=utm +zone=12 +ellps=GRS80 -r <<EOF
```

Reply:
But GRS80 is the default.
No real need to list it.